images/instrumentation/injector/test/scripts/run-tests-within-container.sh

```diff
@@ -42,11 +42,12 @@ run_test_case() {
   command=$2
   expected=$3
   existing_node_options_value=${4:-}
+  existing_otel_resource_attributes=${5:-}
   set +e
   if [ "$existing_node_options_value" != "" ]; then
-    test_output=$(LD_PRELOAD="$injector_binary" TEST_VAR=value NODE_OPTIONS="$existing_node_options_value" node index.js "$command")
+    test_output=$(LD_PRELOAD="$injector_binary" TEST_VAR=value DASH0_NAMESPACE_NAME=my-namespace DASH0_POD_NAME=my-pod DASH0_POD_UID=275ecb36-5aa8-4c2a-9c47-d8bb681b9aff DASH0_CONTAINER_NAME=test-app NODE_OPTIONS="$existing_node_options_value" OTEL_RESOURCE_ATTRIBUTES="${existing_otel_resource_attributes}" node index.js "$command")
   else
-    test_output=$(LD_PRELOAD="$injector_binary" TEST_VAR=value node index.js "$command")
+    test_output=$(LD_PRELOAD="$injector_binary" TEST_VAR=value DASH0_NAMESPACE_NAME=my-namespace DASH0_POD_NAME=my-pod DASH0_POD_UID=275ecb36-5aa8-4c2a-9c47-d8bb681b9aff DASH0_CONTAINER_NAME=test-app OTEL_RESOURCE_ATTRIBUTES="${existing_otel_resource_attributes}" node index.js "$command")
```

Comment:
_nitpick:_ Even with passing an empty string for argument 5, the test case does not accurately reflect the reality of running without OTEL_RESOURCE_ATTRIBUTES being set, because setting an env var to an empty string and not having it set at all is not the same. If you don't mind I will extend this with another check like we already have with `if [ "$existing_node_options_value" != "" ]; then`.

Reply:
done